source/_includes/installation/operating_system.md

```diff
@@ -73,50 +73,51 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
 1. Disable Secure Boot.
 ![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)
 
-1. Save the changes and exit.
+1. Save your changes and exit.
 
-- The BIOS configuration is complete.
+The BIOS configuration is now complete.
 
-As a next step, we need to write the Home Assistant Operating System image to the target boot medium. The HAOS has no integrated installer. This means the Operating System is not copied automatically to the internal disk.
+Next we need to write the Home Assistant Operating System image to the "boot medium", which is the medium your x86-64 hardware will boot from when it is running Home Assistant.
 
-- The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
-- Typically, an internal medium is used for the x86-64 hardware. Examples of internal media are S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC.
-- Alternatively, an external medium can be used to boot HAOS such as a USB SDD (not recommended).
+<div class='note'>
+HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. Etcher.
+</div>
 
-To install the HAOS internally on your x86-64 hardware, there are 2 methods:
+Typically an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively an external medium can be used such as a USB SDD, though this is not recommended.
 
-1. Copying the HAOS disk image from your Desktop computer onto your boot medium (e.g. by using a USB to S-ATA adapter). This is not an option for a non-removable eMMC on your x86-64 hardware, of course.
-To use this method, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
-2. Copying a live operating system (e.g. Ubuntu) onto a USB device. Then, insert this USB device into your x86-64 hardware and start the Ubuntu.
+To write the HAOS image to the boot medium on your x86-64 hardware, there are 2 different methods:
 
-- To use this method, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
+1. Write the HAOS disk image from your desktop computer directly to the boot medium (e.g. using a USB to S-ATA adapter).
+If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal mediums or don't have the necessary adapter, try the next method instead.
```

Comment:
```suggestion
If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal media or don't have the necessary adapter, try the next method instead.
```

Reply:
`mediums` seems more correct here?

Reply:
I cannot say with certainty that mediums would be incorrect in this particular case. I changed it for reasons of consistency, as media is the plural used in other places in this text. I based this on the Microsoft Style Guide entry on [medium, media](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/medium-media), and Merriam Webster [medium](https://www.merriam-webster.com/dictionary/medium). In the context of "something (such as a magnetic disk) on which information may be stored" they use media for the plural.